### DISCUSSION TITLE
Docker-compose update recreates containers after watchtower update

### DISCUSSION THREAD

**Discussion by samcro1967** [containrrr/watchtower#1165] [Q&A]
Everything works fine with watchtower, however when I run docker-compose up in the morning after watchtower has updated containers the previous night, docker-compose recreates any containers that watchtower updated.  Docker-compose does not seem to recognize the containers watchtower updated as being current.  The old images are gone, and the SW version inside the containers is the latest so I am not sure why docker-compose recreates them.  Trying to avoid a second outage on the containers as some of the containers have init scripts that take 5 minutes to run when they are restarted.

Anyone else have a similiar issue and figure out a way to resolve this?

**Comment by simonaronsson** [comment]
It keeps recreating them because it uses a label on the container to track the hash of what is running. Unfortunately, there is currently no good way for us to recalculate that hash. 

So this is a known issue we can't do much about at the moment.
